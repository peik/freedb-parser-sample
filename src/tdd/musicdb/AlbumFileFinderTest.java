package tdd.musicdb;

import static org.junit.Assert.assertTrue;

import java.io.File;

import org.junit.Test;

public class AlbumFileFinderTest {

	@Test
	public void testFindFile() {
		AlbumFileFinder r = AlbumFileFinder.getInstance();
		File file = r.getFile("12025d03.txt");
		System.out.println(file);
		System.out.println(file.getAbsolutePath());
		assertTrue(file.exists());
	}

}
