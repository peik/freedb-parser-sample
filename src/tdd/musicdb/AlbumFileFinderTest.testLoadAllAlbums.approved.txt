------------------------------------------------------------------
Ernest Ray Everett - Single CD	Details
Tracks: 3
Total time: 10:04
Year: 2009
Disc-ID: Country / 12025d03

 
1.	I'm Your Puppet	2:39
2.	Me And Jacky	4:20
3.	If You Don't Like Country	3:05

------------------------------------------------------------------
Beethoven / Beethoven Piano Sonatas	Details
Tracks: 9
Total time: 52:49
Year: 1998
Disc-ID: Classical / 6f0c6509

 
1.	dagio sostenuto	5:19
2.	II. Allegretto	2:02
3.	III. Presto agitato	6:28
4.	 molto e con brio	8:11
5.	II. Adagio contabile	4:58
6.	III. Rondo: Allegro	4:13
7.	ai	8:46
8.	II. Andante con moto	5:56
9.	III. Allegro ma non troppo; Presto	6:56

------------------------------------------------------------------
Nightwish / Angels Fall First	Details
Tracks: 9
Total time: 51:00
Year: 1997
Disc-ID: Metal / 770bf909

 
1.	Elvenpath	4:34
2.	Beauty and the Beast	6:19
3.	The Carpenter	5:53
4.	Astral Romance	5:08
5.	Angels Fall First	5:30
6.	Tutankhamen	5:26
7.	Nymphomaniac Fantasia	4:43
8.	Know Why the Nightingale Sings	4:10
9.	Lappi (Lapland)	9:17

------------------------------------------------------------------
Nightwish / Once	Details
Tracks: 11
Total time: 60:02
Year: 2004
Disc-ID: Symphonic Metal / 840e160b

 
1.	Dark Chest Of Wonders	4:28
2.	Wish I Had An Angel	4:05
3.	Nemo	4:36
4.	Planet Hell	4:38
5.	Creek Mary's Blood	8:29
6.	The Siren	4:45
7.	Dead Gardens	4:28
8.	Romanticide	4:58
9.	Ghost Love Score	10:02
10.	Kuolema Tekee Taiteilijan	3:58
11.	Higher Than Hope	5:35

------------------------------------------------------------------
Van Halen / Van Halen	Details
Tracks: 11
Total time: 35:13
Year: 1978
Disc-ID: Rock / 8608460b

 
1.	Runnin' With The Devil	3:34
2.	Eruption	1:42
3.	You Really Got Me	2:36
4.	Ain't Talkin' 'Bout Love	3:47
5.	I'm The One	3:44
6.	Jamie's Cryin'	3:30
7.	Atomic Punk	3:00
8.	Feel Your Love Tonight	3:41
9.	Little Dreamer	3:21
10.	Ice Cream Man	3:18
11.	On Fire	3:00

------------------------------------------------------------------
Mozart / The Classics: Mozart	Details
Tracks: 12
Total time: 62:32
Disc-ID: Classical / 9c0eac0c

 
1.	The Marriage of Figaro, Overture	4:08
2.	Hallelujah From Exultate Jubilate	2:43
3.	Flute Concerto in D Major, Finale	6:03
4.	Eine Kleine Nachtmusik, 1st Movement	5:53
5.	Violin Concerto in D Major, Last Movement	8:00
6.	Symphony No. 35 in D Major (Haffner), Last Movement	4:07
7.	The Magic Flute, Overture	6:28
8.	Laudate Dominum	5:44
9.	Rondo Alla Turka	3:22
10.	Symphony No. 29 in A Major, 4th Movement	4:20
11.	Horn Concerto No. 3 in E Flat Major, 1st Movement	7:04
12.	Symphony No. 41 in C (Jupiter), 1st Movement	4:40

------------------------------------------------------------------
Madonna / Ray of light	Details
Tracks: 12
Total time: 61:33
Disc-ID: Pop / 9e0e740c

 
1.	Drowned world / substitue for love	5:10
2.	Swim	5:02
3.	Ray of light	5:22
4.	Candy perfume girl	4:38
5.	Skin	6:23
6.	Nothing really matters	4:28
7.	Sky fits heaven	4:49
8.	Shanti / ashtangi	4:31
9.	Frozen	6:14
10.	The power of goodbye	4:14
11.	To have and not to hold	5:24
12.	Mer girl	5:18

------------------------------------------------------------------
Adele / 21	Details
Tracks: 13
Total time: 54:40
Year: 2011
Disc-ID: Pop / b20cd40d

 
1.	Rolling in the deep	3:28
2.	Rumour has it	3:49
3.	Turning tables	3:43
4.	Don't you remember	4:10
5.	Set fire to the rain	4:03
6.	He won't go	4:01
7.	Take it all	4:37
8.	I'll be waiting	3:48
9.	One and only	4:01
10.	Lovesong	5:48
11.	Someone like you	5:16
12.	If it hadn't been for love	4:47
13.	Hiding my heart	3:09

------------------------------------------------------------------
Adele / 21	Details
Tracks: 13
Total time: 54:33
Year: 2011
Disc-ID: Pop / c00ccd0d

 
1.	Rolling In The Deep	3:49
2.	Rumour Has It	3:43
3.	Turning Tables	4:10
4.	Don't You Remember	4:03
5.	Set Fire To The Rain	4:01
6.	He Won't Go	4:37
7.	Take It All	3:48
8.	I'll Be Waiting	4:01
9.	One And Only	5:48
10.	Lovesong	5:16
11.	Someone Like You	4:43
12.	If It Hadn't Been For Love	3:08
13.	Hiding My Heart	3:26

------------------------------------------------------------------
Madonna / The Immaculate Collection	Details
Tracks: 17
Total time: 73:59
Year: 2005
Disc-ID: other / d1115d11

 
1.	Lucky Star	4:08
2.	Borderline	3:39
3.	Like A Virgin	4:02
4.	Material Girl	3:13
5.	Crazy For You	3:55
6.	Into The Groove	3:47
7.	Live To Tell	4:11
8.	Papa Don't Preach	5:20
9.	Open Your Heart	4:13
10.	La Isla Bonita	3:53
11.	Like A Prayer	3:50
12.	Express Yourself	5:53
13.	Cherish	4:06
14.	Vogue	3:54
15.	Justify My Love	5:21
16.	Rescue Me	5:02
17.	Holiday	5:32

------------------------------------------------------------------
Ray Charles / Ray Charles Sings The Blues	Details
Tracks: 18
Total time: 49:22
Year: 1995
Disc-ID: Blues / fd0b9912

 
1.	C.C. Rider (See-See Rider Blues)	2:42
2.	Going Down To The River	3:02
3.	Ray Charles Blues	2:26
4.	Someday (You're Gonna Miss Me)	3:03
5.	How Long	2:32
6.	Blues Is My Middle Name	3:09
7.	Baby, Won't You Come Back Home	2:51
8.	Rockin' Chair Blues	2:45
9.	Goin' Down Slow	2:41
10.	Sentimental Blues	2:25
11.	If I Give You My Love	2:31
12.	I Wonder Who's Kissing Her	2:27
13.	All To Myself Alone	2:01
14.	Falling Blues	2:50
15.	I'm Glad For Your Sake	2:46
16.	Walkin' And Talkin'	3:07
17.	Baby, Let Me Hold Your Hand	3:11
18.	Alone In The City Blues	2:53

